lib/measured/conversion.rb

```diff
@@ -69,6 +70,11 @@ def add_new_unit(unit_name, aliases:, value: nil, base: false)
     @units << unit
     @base_unit = unit if base
 
+    unit.names.each do |name|
+      @unit_name_to_unit[name] = unit
+      @unit_name_to_unit[name.downcase] = unit
```

Comment:
Since `case_sensitive` can't be changed after initialization, we can definitely branch and only add the things we need. Will update!